### ISSUE TITLE
test.vcf fails

### ISSUE THREAD

**Issue by jwagnerhki** [rbutleriii/Clinotator#10]
I wanted to try test.vcf to see what the expected VCF format might be, because VCF produced by Platypus also causes a KeyError in Clinotator. However, in v1.3.1 and trunk, even the included test.vcf causes a KeyError. The trace is below:

$ clinotator.py --long-log -o clinotator -e my.email@somewhere.com -t vcf test.vcf
Traceback (most recent call last):
  File "/home/oper/jwagner/Clinotator/clinotator/clinotator.py", line 193, in <module>
    main()
  File "/home/oper/jwagner/Clinotator/clinotator/clinotator.py", line 184, in main
    variation.query_parsing(variant_objects, query_results)
  File "/home/oper/jwagner/Clinotator/clinotator/variation.py", line 30, in query_parsing
    variant = VariationClass(variationreport)
  File "/home/oper/jwagner/Clinotator/clinotator/variation.py", line 107, in __init__
    self.VID = variationreport.attrib['VariationID']
KeyError: 'VariationID'

**Comment by rbutleriii** [comment]
Update: I checked the output and the issue is that the ClinVar xml at NCBI has fundamentally changed its format. The normal location for the Entrez query returns an empty result.
```
<?xml version="1.0" encoding="UTF-8" ?>
                        
<ClinVarResult-Set><set/></ClinVarResult-Set>
```
I will investigate and see how the data has been changed, and work on a fix.

**Comment by rbutleriii** [comment]
Just released version 1.4.0, which parses the new ClinVar XML format. Should work now.
